src/lib.rs

```diff
@@ -88,14 +90,12 @@ impl fmt::Display for ParseError {
 impl ParseError {
     /// Returns `true` if this error is `NoLeadingBackslash`; otherwise returns
     /// `false`.
-    #[must_use]
```

Comment:
Why was this needed?

Reply:
Oh, I left a note on your last comment about `#[must_use]` on `from_static`. Basically the lint said there could be many false positives so I just rolled back adding them and disabled the lint.

I can add them back with an exception on `from_static`. `cargo clippy --fix` will do it for us, even, if the lint is enabled.